app/runner.go

```diff
@@ -39,6 +41,9 @@ type MultiRunner struct {
 	// before stopping execution and returning a timeout error instead of exiting gracefully.
 	// If ExitWait is nil, Run execution will always block until all Runners have exited.
 	ExitWait *time.Duration
+	// HelthCheckInterval is the duration at which the runner will periodically run the registered health checks. The
+	// cached result is what's returned by the ready endpoint.
+	HealthCheckInterval time.Duration
```

Comment:
Since this isn't used in the MultiRunner, I think it's probably better to track it externally (in the caller of the health checks).